src/components/application.js

```diff
@@ -83,7 +97,12 @@ export default class Application extends Component {
             />
           </div>
         </div>
-        <Dock title={p.title}>
+        <Dock>
+          <div className='TauiTitle'>
+            <Icon icon='map' />
+            <span>{p.title || message('Title')}</span>
+            {p.info && <Icon icon='info-circle' onClick={this._showInfo} />}
```

Comment:
In Chrome on my machine, the cursor does not indicate this icon is clickable on hover. Is there a way to more clearly identify this info icon as a link, and maybe right-justify it to give the title more space?

Reply:
Will do 👍

Reply:
Changed in https://github.com/conveyal/taui/pull/127/commits/32f454bfafe94eef970507b00f92cbc5d2439246